<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInit</class>
 <widget class="QDialog" name="DialogInit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Initial Conditions</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1" colspan="2">
    <widget class="QCheckBox" name="checkBoxEngineOn">
     <property name="text">
      <string>Engine On</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Aircraft Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboAircrafts</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboAircrafts"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Preset Location:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboLocations</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboLocations">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelInitLat">
     <property name="text">
      <string>Latitude:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinInitLat</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="spinInitLat">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>-1.570796000000000</double>
     </property>
     <property name="maximum">
      <double>1.570796000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="ComboUnitsAngle" name="comboInitLat"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelInitLon">
     <property name="text">
      <string>Longitude:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinInitLon</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="spinInitLon">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>-3.141592000000000</double>
     </property>
     <property name="maximum">
      <double>3.141592000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="ComboUnitsAngle" name="comboInitLon"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelInitAlt">
     <property name="text">
      <string>Altitude AGL:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinInitAlt</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="spinInitAlt">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="ComboUnitsLength" name="comboInitAlt"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelInitPsi">
     <property name="text">
      <string>Heading:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinInitPsi</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="spinInitPsi">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>6.280000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="ComboUnitsAngle" name="comboInitPsi"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelInitIAS">
     <property name="text">
      <string>Airspeed:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinInitIAS</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="spinInitIAS">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="ComboUnitsVelocity" name="comboInitIAS"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelInitDate">
     <property name="text">
      <string>Date UTC:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>dateInit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDateEdit" name="dateInit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelInitTime">
     <property name="text">
      <string>Time UTC:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>timeInit</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QTimeEdit" name="timeInit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
     <property name="time">
      <time>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>351</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboUnitsAngle</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsAngle.h</header>
  </customwidget>
  <customwidget>
   <class>ComboUnitsVelocity</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsVelocity.h</header>
  </customwidget>
  <customwidget>
   <class>ComboUnitsLength</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsLength.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboAircrafts</tabstop>
  <tabstop>comboLocations</tabstop>
  <tabstop>spinInitLat</tabstop>
  <tabstop>comboInitLat</tabstop>
  <tabstop>spinInitLon</tabstop>
  <tabstop>comboInitLon</tabstop>
  <tabstop>spinInitAlt</tabstop>
  <tabstop>comboInitAlt</tabstop>
  <tabstop>spinInitPsi</tabstop>
  <tabstop>comboInitPsi</tabstop>
  <tabstop>spinInitIAS</tabstop>
  <tabstop>comboInitIAS</tabstop>
  <tabstop>dateInit</tabstop>
  <tabstop>timeInit</tabstop>
  <tabstop>checkBoxEngineOn</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogInit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogInit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
